lang: [it] Added the remaining translations for the Italian language

FrancescoCiannavei opened the pull request:
**Description**
I added the remaining translations for the Italian language.
I noticed some translations were missing, I translated all the remaining strings (I am Italian).

I took some light liberties on the following translations:

1. bannedUser => I translated "**you are currently banned**" to "**l'account è stato disabilitato**" which means "**the account has been disabled**"
2. logOutBannedUser => I translated "**you have been banned.**" to "**l'account è stato disabilitato.**" which means "**the account has been disabled**"
3. invalidEmail => "**the email address "{0}" matches the email on record.**" to "**l'indirizzo email "{0}" corrisponda ad un indirizzo registrato.**" which means "**the "{0}" email address matches with a registered one.**"
4. I kept "**Token**" as such, we don't have a word for that

Please let me know if you prefer a more more literal translation.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

FrancescoCiannavei committed:
lang: added the remaining translations for the italian language

datamweb commented:
> 4\. I kept "**Token**" as such, we don't have a word for that

So, please update the test using `Auth.token`:
```php
final class ItalianTranslationTest extends AbstractTranslationTestCase
{
    protected array $excludedLocaleKeyTranslations = [
        'Auth.password',
        'Auth.login',
    ];
}
```

FrancescoCiannavei committed:
lang: excluded locale keys in ItalianTranslationTest

As asked in this comment: https://github.com/codeigniter4/shield/pull/1282#issuecomment-3124711701
I Added "Auth.token" to the "ItalianTranslationTest" class.
Just like in the "CzechTranslationTest" class I removed "Auth.password" and "Auth.login" from the "$excludedLocaleKeyTranslations" array

FrancescoCiannavei commented:
@datamweb I did it.
Also, just like in the "CzechTranslationTest" class I removed "Auth.password" and "Auth.login" from the "$excludedLocaleKeyTranslations" array.
For reference, this is the class: https://github.com/codeigniter4/shield/blob/develop/tests/Language/CzechTranslationTest.php

FrancescoCiannavei committed:
lang: added again "Auth.password" and "Auth.login" to the $excludedLocaleKeyTranslations array

FrancescoCiannavei commented:
Ok, I made a stupid mistake, I added `Auth.password` and `Auth.login` to the `$excludedLocaleKeyTranslations` array again.
I ran the tests on my machine using `php vendor/bin/phpunit` and it passed all tests.

datamweb commented:
@Franky5831 We had a difference of opinion regarding a security report in the CI4 framework, which caused a bit of a delay. Fortunately, after follow-ups and clarification with the team, the issue has been resolved.

Thanks again for taking the time to submit the PR — we truly appreciate your contribution. 🙏
